rustfmt-core/rustfmt-lib/tests/source/cfg_if/lib.rs

```diff
@@ -10,6 +10,7 @@
 //! * `mips64`: [`is_mips64_feature_detected`]
 //! * `powerpc`: [`is_powerpc_feature_detected`]
 //! * `powerpc64`: [`is_powerpc64_feature_detected`]
+// rustfmt-recursive: true
```

Comment:
I _think_ the system/idempotence checks need the inline config options to be at the top of the files as well

Reply:
Thanks, @calebcartwright ! Even if the config options can 'survive' being down below, I also like how having them at the top matches with their location in other files already. I've gone ahead and moved them up there.

I'll wait on topecongiro's feedback on the particular files we can take off the whitelist, then make those changes as well (feel free to weigh in there too if you've got strong opinions).